test/syscall/wt2336_base/base.run

```diff
@@ -46,15 +46,15 @@ SYSTEM("Darwin");
 #define FTRUNCATE(fd, len)			/* do nothing */
 #endif
 
-TRACE("close,fdatasync,fsync,ftruncate,open,pwrite,rename");
+TRACE("close,fdatasync,fsync,ftruncate,open,pwrite64,rename");
 RUN("");
 ...
 OUTPUT("--------------wiredtiger_open");
 // lock == 3
 lock = open("./WiredTiger.lock", O_RDWR|O_CREAT|O_CLOEXEC, 0666);
-pwrite(lock, "WiredTiger lock file\n", 0x15, 0x0);
+pwrite64(lock, "WiredTiger lock file\n", 0x15, 0x0);
```

Comment:
I've made a halfway change for this; the code now simply converts all the input pwrite64's to pwrite's using a regex. This made testing work on both OSX and on Linux (the flavours that use pwrite64 at-least).